docs/reference/commandline/run.md

```diff
@@ -15,137 +15,106 @@ Aliases:
-  -h, --hostname string               Container host name
-      --init                          Run an init inside the container that forwards signals and reaps processes
-  -i, --interactive                   Keep STDIN open even if not attached
-      --io-maxbandwidth string        Maximum IO bandwidth limit for the system drive (Windows only)
-                                      (Windows only). The format is `<number><unit>`.
-                                      Unit is optional and can be `b` (bytes per second),
-                                      `k` (kilobytes per second), `m` (megabytes per second),
-                                      or `g` (gigabytes per second). If you omit the unit,
-                                      the system uses bytes per second.
-                                      --io-maxbandwidth and --io-maxiops are mutually exclusive options.
-      --io-maxiops uint               Maximum IOps limit for the system drive (Windows only)
-      --ip string                     IPv4 address (e.g., 172.30.100.104)
-      --ip6 string                    IPv6 address (e.g., 2001:db8::33)
-      --ipc string                    IPC namespace to use
-      --isolation string              Container isolation technology
-      --kernel-memory string          Kernel memory limit
-  -l, --label value                   Set meta data on a container (default [])
-      --label-file value              Read in a line delimited file of labels (default [])
-      --link value                    Add link to another container (default [])
-      --link-local-ip value           Container IPv4/IPv6 link-local addresses (default [])
-      --log-driver string             Logging driver for the container
-      --log-opt value                 Log driver options (default [])
-      --mac-address string            Container MAC address (e.g., 92:d0:c6:0a:29:33)
-  -m, --memory string                 Memory limit
-      --memory-reservation string     Memory soft limit
-      --memory-swap string            Swap limit equal to memory plus swap: '-1' to enable unlimited swap
-      --memory-swappiness int         Tune container memory swappiness (0 to 100) (default -1)
-      --mount value                   Attach a filesystem mount to the container (default [])
-      --name string                   Assign a name to the container
-      --network-alias value           Add network-scoped alias for the container (default [])
-      --network string                Connect a container to a network
-                                      'bridge': create a network stack on the default Docker bridge
-                                      'none': no networking
-                                      'container:<name|id>': reuse another container's network stack
-                                      'host': use the Docker host network stack
-                                      '<network-name>|<network-id>': connect to a user-defined network
```

Comment:
Looks like there were some manually written additions here (describing the different network modes) that are not printed on the CLI itself. Those were removed now (should we make the cli itself print these?)